source/reference/minio-mc/mc-support-logs-status.rst

```diff
@@ -0,0 +1,50 @@
+==========================
+``mc support logs status``
+==========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. mc:: mc support logs status
+
+Description
+-----------
+
+Use the :mc-cmd:`mc support logs status` command to output whether the specified ALIAS is currently uploading logs to |subnet|.
```

Comment:
currently uploading, or just whether it's enabled?

Reply:
D'oh. Whether or not enabled. Nice catch.